src/share/poudriere/jail.sh

```diff
@@ -1008,6 +1092,9 @@ while getopts "biJ:j:v:a:z:m:nf:M:sdkK:lqcip:r:uU:t:z:P:S:Dx" FLAG; do
 		x)
 			XDEV=1
 			;;
+		X)
+			XDEV=1
```

Comment:
Er, -X does the same thing as -x in your patch. I don't see the need for a new flag anyway. -x should be sufficient for all cases. Just revert the getops/X)/usage changes.